src/themes/Treedip/DonationForm/SubComponents/SelectCustomAmountRadioBubble.scss

```diff
@@ -0,0 +1,105 @@
+@use 'src/components/DonationForm/SubComponents/SelectCustomAmount';
+@use '../../variables/colors';
+
+$height: 16px !default;
+$width: 16px !default;
+$margin: -6px 0 0 0 !default;
+$background-color: transparent !default;
+$border-radius: 0 !default;
+$input-container-border-radius: 0 !default;
+$border: 1px solid #bbbbbb !default;
+$border-radio: 6px solid colors.$primary !default;
+$box-shadow: none !default;
+$background-color-radio: transparent !default;
+
+.wmde-banner {
+	@include SelectCustomAmount.layout;
+
+	/*
+	The layout settings of this element should be similar to wmde-banner-select-group-option
+	because the custom amount is presented as an "option" inside the amount select group
+	In the future, we might use tools of the CSS framework (e.g. SCSS mixins)
+	to keep the styles in sync
+	 */
+	&-select-custom-amount {
```

Comment:
If you compare the hierarchy of elements in the amount selection (option elements vs custom amount element), you can see that the hierarchy is different. This class is the equivalent to the `label` element of the regular [SelectGroupRadioBubble](https://github.com/wmde/fundraising-banners/pull/228/files#diff-f5284e8cf5d1a22a5e6e669a7858c4b9e90c528685806b00dd4756114b111c82R20-R47). My approach here would be to try and apply the classes from the `label` element here (and the equivalent subclasses).
If that doesn't work correctly due to flexbox issues (in regular options the `.wmde-banner-option` is a container that has a flex attribute and determines the size of the children and that's missing here), then I'd recommend adding a div with the `wmde-banner-option` around the custom amount component.